### DISCUSSION TITLE
How when in container open link (new tab) without container (middle mouse click) ?

### DISCUSSION THREAD

**Discussion by geextahslex** [mozilla/multi-account-containers#2677] [Q&A]
Hi, how can I achieve this behaviour?
I know about `right click > Open Link in New Container Tab > No Container` but this is inconvenient 

Thank you :)

**Comment by kelimuttu** [comment]
I'm not aware of any current shortcut to do this. Would you please open [a bug report](https://github.com/mozilla/multi-account-containers/issues/new/choose)?
